### PR TITLE
package pwlf

### PR DESCRIPTION
**Pull request by h-vetinari**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

Naming follows package on [PyPI](https://pypi.org/project/pwlf/).

@cjekel, I'm packaging `pwlf` for conda-forge, you're welcome to maintain the recipe with me if you want!

### CODE REVIEW

File: recipes/pwlf/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (synapticarbors) [inline recipes/pwlf/LICENSE:1]: You can delete this file since it's already included in the source distribution

**Comment by h-vetinari** [comment]
Seems that [`version.py`](https://github.com/cjekel/piecewise_linear_fit_py/blob/master/pwlf/version.py) depends on `setuptools`. This is not ideal (and could be patched out in the recipe), but for the moment I'm adding `setuptools` as a run dependency.

File: recipes/pwlf/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - pydoe >=0.3.8
+    - numpy
+    - scipy >=1.2.0
+    - setuptools
```
Comment (h-vetinari) [inline recipes/pwlf/meta.yaml:26]: See [this](https://github.com/conda-forge/staged-recipes/pull/11336#issuecomment-617163048) comment.

**Comment by cjekel** [comment]
@h-vetinari Thanks for this! Feel free to ping me if something comes up. 

> Seems that version.py depends on setuptools. This is not ideal (and could be patched out in the recipe), but for the moment I'm adding setuptools as a run dependency.

This was one of those dirty tricks to get a single source version file. [Python docs](https://packaging.python.org/guides/single-sourcing-package-version/) have some other suggestions. I'm open to switching to another method if it causes issues here.

**Comment by h-vetinari** [comment]
Hey @cjekel, thanks for the response! I think that, from the variants in the link you posted, Number 5 would be the cleanest solution. It would certainly help us have less package pollution in an environment if we can remove `setuptool` from the run requirements.

**Comment by cjekel** [comment]
@h-vetinari so pwlf==2.0.1 has just been released on pypi. Only change is the remove of setuptools back to disutils.core with these new requirements
```
        "numpy >= 1.14.0",
        "scipy >= 1.2.0",
        "pyDOE >= 0.3.8",
        'importlib-metadata ~= 1.0 ; python_version < "3.8"',
```

**Comment by h-vetinari** [comment]
@cjekel, cool thanks!
I already got an update-PR from the bot: https://github.com/conda-forge/pwlf-feedstock/pull/1

**Pull request merged**